frontend/src/components/ImplementationReportView/EmbedBadges.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import { Implementation } from "../../data/parseReportData";
+import CopyToClipboard from "../CopyToClipboard";
+import Dialect from "../../data/Dialect";
+import { complianceBadgeFor, versionsBadgeFor } from "../../data/Badge";
+import { badgeFormatOptions, BadgeFormatOption } from "./BadgeFormats";
+
+const EmbedBadges: React.FC<{
+  implementation: Implementation;
+}> = ({ implementation }) => {
+  const [activeTab, setActiveTab] = useState("URL");
+  const [activeBadge, setActiveBadge] = useState("JSON Schema Versions");
+  const [badgeURI, setBadgeURI] = useState(
+    versionsBadgeFor(implementation).href(),
+  );
+
+  const handleSelectBadge = (badgeName: string, URI: string): void => {
+    setActiveBadge(badgeName);
+    setBadgeURI(URI);
+  };
+
+  const handleSelectTab = (tabKey: string | null) => {
+    if (tabKey) {
+      setActiveTab(tabKey);
+    }
+  };
+
+  const results = Object.entries(implementation.results).sort((a, b) => {
+    return (
+      Dialect.withName(b[0]).firstPublicationDate.getTime() -
+      Dialect.withName(a[0]).firstPublicationDate.getTime()
+    );
+  });
+
+  return (
+    <div className="container dropdown px-0 col-12">
+      <button
+        className="btn btn-sm btn-success dropdown-toggle"
+        type="button"
+        data-bs-toggle="dropdown"
+        data-bs-auto-close="outside"
+        style={{ width: "80px" }}
+      >
+        Badges
+      </button>
+      <ul className="dropdown-menu mx-auto mb-3">
+        <li>
+          <div>
+            <p className="text-center fw-semibold pt-2 pb-1 px-1 fs-6 mb-2">
+              Generate Bowtie Badge
+            </p>
+            <div className="dropdown d-flex flex-column justify-content-center align-items-center px-2">
+              <label className="pb-1" htmlFor="dropdownMenuButton">
+                Available Badges
+              </label>
+              <button
+                className="btn btn-sm btn-primary dropdown-toggle mx-auto"
+                type="button"
+                id="dropdownMenuButton"
+                data-bs-toggle="dropdown"
+                aria-expanded="false"
+              >
+                {activeBadge === "JSON Schema Versions"
```

Comment:
You still have vestiges here of `if` statements checking what badge we're showing. Try to remove them!